examples/SageMaker_SDK_Tutorial.ipynb

```diff
@@ -860,9 +860,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "conda_mxnet_p36",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "conda_mxnet_p36"
+   "name": "python3"
   },
```

Comment:
You probably want to leave these set to one of the default kernels available on SageMaker instances (same for the Python version a few lines below), so that users don't have to select it themselves: you can try out on an instance, if you upload the notebook as it is, Jupyter will probably complain that the kernel is not there and you should select a new one.